### PR TITLE
feat: log incorrect code as failed login

### PR DESCRIPTION
**Pull request by danielgent**
Matching Auth0

image

I'm not sure what will happen here because our logger only logs when `!!response.ok` :thinking:

### CODE REVIEW

File: src/tsoa-middlewares/logger.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export enum LogTypes {
   SUCCESS_LOGOUT = "slo",
   SUCCESS_SILENT_AUTH = "ssa",
   SUCCESS_CROSS_ORIGIN_AUTHENTICATION = "scoa",
+  FAILED_LOGIN_WRONG_PASSWORD = "flwp",
```
Comment (markusahlstrand) [inline src/tsoa-middlewares/logger.ts:17]: I love these abbreviations.. makes you sound so smart!

File: src/routes/tsoa/authenticate.ts [hunk 2]
```diff
@@ -72,6 +73,9 @@ export class AuthenticateController extends Controller {
       const otps = await env.data.OTP.list(client.tenant_id, email);
       const otp = otps.find((otp) => otp.code === body.otp);
 
+      // TODO - release this and see what happens...
```
Comment (markusahlstrand) [inline src/routes/tsoa/authenticate.ts:76]: Nice todo :) I guess we will...

**Review by markusahlstrand — APPROVED** [review]
LGTM!

File: src/tsoa-middlewares/logger.ts (hunk 1 above)
Comment (danielgent) [reply src/tsoa-middlewares/logger.ts:17]: I didn't start off these abbreviations! It was also chatgpt generated 😎

File: src/routes/tsoa/authenticate.ts (hunk 2 above)
Comment (danielgent) [reply src/routes/tsoa/authenticate.ts:76]: I'll actually remove this comment though!!!

```suggestion
```

**Pull request merged**